### ISSUE TITLE
false positive with no-unused-module and export syntax

### ISSUE THREAD

**Issue by ulrik59** [benmosher/eslint-plugin-import#1544]
It seems that the rule `no-unused-module` is returning false positive result with `export { default } from '...'` syntax (which is a valid syntax given [MDN documentation](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export)).

**Example:**
Given I have 3 files:

_./myModule1/data.js_
```js
const getData = () => {
  // awesome things
}

export default getData;
```

_./myModule1/index.js_
```js
export { default } from './data';
```

_./myModule2.js_
```js
import getData from './myModule1';
```
Then I get this error:
```
/home/myName/myAwesomeProject/myModule1/index.js
  1:1  error  exported declaration 'default' not used within other modules  import/no-unused-modules
```
Expect behaviour is there should be no error.

**Comment by Ephem** [comment]
This seems to have been fixed by @richardxia in #1640 which was released in [v2.20.2](https://github.com/benmosher/eslint-plugin-import/blob/master/CHANGELOG.md#2202---2020-03-28).

Hopefully that helps and this issue can be closed! 😄
